modules/graphql_core/graphql_core.info.yml

```diff
@@ -1,7 +1,6 @@
 name: GraphQL Core
 type: module
 description: 'Provides type system plugins and derivers on behalf of core modules.'
-package: GraphQL
-core: 8.x
+package: Testing
```

Comment:
Why? This is not a 'Testing' package - it is a normal submodule that is meant to be installed by users.